get-beam.sh

```diff
@@ -12,6 +12,15 @@
                 command -v "$@" >/dev/null 2>&1
         }
 
+	if [ ! -d "$install_dir" ]; then
+                echo "Install directory doesn't exist. Creating now."
```

Comment:
Might be useful to print out the install dir here, so users know what the issue is if this fails due to a permissions error or something. For example: "Installation directory: /usr/local/bin/ does not exists. Creating it now..."

Reply:
Oh it looks like you are printing in the error message. I still think it might not hurt to have it here as well.